BuildingMaterials/BuildingMaterials/Models/MaterialModel.cs

```diff
@@ -0,0 +1,21 @@
+using System.Text.Json.Serialization;
+
+namespace BuildingMaterials.Models;
+
+public class MaterialModel
+{
+    [JsonPropertyName("id")] public int Id { get; set; }
```

Comment:
You can set CamelCase naming convention on class level `[JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]`